docs/2.guide/3.going-further/6.nuxt-app.md

```diff
@@ -30,6 +30,8 @@ export function useMyComposable () {
 }
 ```
 
+If your composable does not always need `nuxtApp` or you simply want to check if it is present or not, since [`useNuxtApp()`](/docs/api/composables/use-nuxt-app) throws an exception, you can use [`tryUseNuxtApp`](/docs/api/composables/use-nuxt-app#tryusenuxtapp) instead.
```

Comment:
```suggestion
If your composable does not always need `nuxtApp` or you simply want to check if it is present or not, since [`useNuxtApp`](/docs/api/composables/use-nuxt-app) throws an exception, you can use [`tryUseNuxtApp`](/docs/api/composables/use-nuxt-app#tryusenuxtapp) instead.
```